fv3core/fv3core/decorators.py

```diff
@@ -8,8 +8,8 @@
 import fv3core._config as spec
 import fv3core.utils
 import fv3core.utils.grid
-from fv3core.utils.stencil import FrozenStencil, StencilFactory
-from fv3core.utils.typing import Index3D
+from pace.dsl.stencil import FrozenStencil, StencilFactory
+from pace.dsl.typing import Index3D
```

Comment:
I'm open to this being a different PR< but we might want to delete this file. state_inputs and get_namespace are no longer used, and get_stencils_with_varied_bounds probably makes sense to get moved in this PR to the pace.dsl space? It could be a feature that physics uses (or not, there are usually other workarounds)

Reply:
Yes, will move `get_stencils_with_varied_bounds` to `pace.dsl` in this PR. In light of Christmas cleaning, I'll also delete the decorators file with `state_inputs` and `get_namespace`.

Reply:
Okay, we found that `fvsubgridz` still uses argspec. Will clean that out in a different PR.